visual-dotnet/SauceLabs.Visual/VisualClient.cs

```diff
@@ -26,6 +29,12 @@ public class VisualClient : IDisposable
         public bool CaptureDom { get; set; } = false;
         private readonly ResiliencePipeline _retryPipeline;
 
+        public string? CurrentTestClass { get; set; }
+        public string? CurrentTestName { get; set; }
+        private string? _previousTestClass = null;
+
+        #region Create
```

Comment:
smal note aboute regions:

They're considered as anti-pattern because if you want to use them it means your class it to big. 

I'm ok but maybe we should consider moving some business logic from this to small testable classes?

Reply:
Let's remove it 👍  It was mainly allowing to group collapsing parts.